README.md

```diff
@@ -23,31 +23,35 @@ and renders declaratively using `lit-html`.
     be performed when a property is set, for example validation. Call `requestUpdate(name, oldValue)`
     in the setter to trigger an update and use any configured property options.
 
-    Properties can be given an options argument which is an object that describes how to
+    Properties can be given an `options` argument which is an object that describes how to
     process the property. This can be done either in the `@property({...})` decorator or in the
     object returned from the `properties` getter, e.g. `static get properties { return { foo: {...} }`.
 
     Property options include:
 
     * `attribute`: Indicates how and whether the property becomes an observed attribute.
-    If the value is `false`, the property is not added to `observedAttributes`.
-    If true or absent, the lowercased property name is observed (e.g. `fooBar` becomes `foobar`).
+    If the value is `false`, the property is not added to the static `observedAttributes` getter.
+    If `true` or absent, the lowercased property name is observed (e.g. `fooBar` becomes `foobar`).
     If a string, the string value is observed (e.g `attribute: 'foo-bar'`).
     * `type`: Indicates how to serialize and deserialize the attribute to/from a property.
     If this value is a function, it is used to deserialize the attribute value
-    a the property value. If it's an object, it can have keys for `fromAttribute` and
+    to the property value. If it's an object, it can have keys for `fromAttribute` and
     `toAttribute` where `fromAttribute` is the deserialize function and `toAttribute`
     is a serialize function used to set the property to an attribute. If no `toAttribute`
-    function is provided and `reflect` is set to `true`, the property value is set
-    directly to the attribute.
-    * `reflect`: Indicates if the property should reflect to an attribute.
-    If `true`, when the property is set, the attribute is set using the
-    attribute name determined according to the rules for the `attribute`
-    property option and the value of the property serialized using the rules from
-    the `type` property option.
-    * `hasChanged`: A function that indicates if a property should be considered
-    changed when it is set. The function should take the `newValue` and `oldValue`
-    and return `true` if an update should be requested.
+    function is provided and `reflect` is set to `true`, the attribute value is
+    set directly using the property value, thus using the property value's associated
+    `toString` method. In the case that is missing, serialization will throw.
+    * `reflect`: Indicates whether setting this property will be reflected in the
```

Comment:
nit:

> whether setting this property will be reflected

That reads a little weird to me. I like the original wording, but maybe with the detail added here:

"whether the property should reflect to its associated attribute (as determined by the `attribute` option)"